### PR TITLE
feat! Turn off survey link for merged/closed PRs

### PR DESCRIPTION
**Pull request by sarina**
Per discussion in standup, let's disable this for now until we have a better story around looking at the data and only posting on first-timers PRs

### CODE REVIEW

**Comment by sarina** [comment]
@kdmccormick is this right?

**Review by kdmccormick — APPROVED** [review]
🔥 🙌🏻 🦘

File: openedx_webhooks/tasks/pr_tracking.py [hunk 1]
```diff
@@ -291,8 +291,9 @@ def desired_support_state(pr: PrDict) -> PrDesiredInfo:
         if state == "reopened":
             desired.bot_comments_to_remove.add(BotComment.SURVEY)
 
-        if state in ["closed", "merged"]:
-            desired.bot_comments.add(BotComment.SURVEY)
+#        # temp: Disable survey link on pull requests
```
Comment (kdmccormick) [inline openedx_webhooks/tasks/pr_tracking.py:294]:
```suggestion
#        # temp: Disable survey link on pull requests
#        # https://github.com/openedx/openedx-webhooks/issues/259
```

**Review by kdmccormick — APPROVED** [review]

**Comment by kdmccormick** [comment]
@sarina I merged your PR, the only issue was commitlint wanted `feat!` to be `feat!:`.

I'm not sure if this will autodeploy or if there's some step you need to do in order to deploy it.

**Comment by kdmccormick** [comment]
@sarina whoops, I totally missed that there were unit test failures in alongside the commitlint failure 😅 Could you give https://github.com/openedx/openedx-webhooks/pull/301 a review?

**Pull request merged**